randovania/games/dread/logic_database/Dairon.txt

```diff
@@ -991,6 +999,15 @@ Extra - asset_id: collision_camera_012
   * Extra - left_shield_def: None
   * Extra - right_shield_entity: {EMPTY}
   * Extra - right_shield_def: None
+  > Door to Early Grapple Access
+      All of the following:
+          Morph Ball
+          Any of the following:
+              Varia Suit
+              Heat/Cold Runs (Intermediate) and Heat Damage ≥ 450
+          Any of the following:
+              Flash Shift or Use Spin Boost
+              Cross Bomb and Movement (Beginner)
```

Comment:
Not sure why we never had this as just trivial, since you just fall down onto the slope and jump up to the tunnel. Cross Bomb can be removed for the movement requirement.

Reply:
Good point, I'll remove this one. We already had beginner movement for making the left jump in this this direction, so it doesn't make a specific impact if the right one is also considered  beginner movement or would be trickless.